CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 ==================
 ### Changed
 ### Added
+* Support Node.js v13
```

Comment:
@csvn could you remove this line please so people don't think that a new version of Canvas is required to use Node.js v13? Then we can merge this :)

Reply:
Sure! Removed any changes to `CHANGELOG.md`.